diesel/src/mysql/connection/stmt/iterator.rs

```diff
@@ -59,6 +62,26 @@ impl<'a> Row<Mysql> for MysqlRow<'a> {
     fn next_is_null(&self, count: usize) -> bool {
         (0..count).all(|i| self.binds.field_data(self.col_idx + i).is_none())
     }
+
+    fn column_count(&self) -> usize {
+        self.binds.len()
+    }
+
+    fn column_name(&self) -> Option<&str> {
+        let metadata = self.stmt.metadata().expect("Failed to get metadata");
```

Comment:
I think, you should specify here from what backend this message, because when user will get that message in multidb application, it will be very cryptic